modules/file-rotation/file-rotation-grammar.ym

```diff
@@ -80,6 +81,7 @@ file_rotation_options
 
 file_rotation_option
   : KW_SIZE '(' nonnegative_integer ')' { file_rotation_set_size(last_file_rotation_plugin, $3); }
+  | KW_INTERVAL '(' string ')' { log_rotation_set_interval(last_file_rotation_plugin, $3); free($3); }
```

Comment:
The PR still can not be built. `log_rotation_set_interval` -> `file_rotation_set_interval`

Reply:
The file where I made changes couldn't be pushed due to git issues.